VEHICLE/SetTrainCruiseSpeed.md

```diff
@@ -8,8 +8,16 @@ ns: VEHICLE
 cs_type(Any) void SET_TRAIN_CRUISE_SPEED(Vehicle train, float speed);
 ```
 
+Used to control train speed, can be used to start and stop its movement as well.
+
 ## Parameters
 * **train**: 
 * **speed**: 
 
 ## Return value
+
+## Examples
+```lua
```

Comment:
need train to be initialized here

Reply:
Fixed

Reply:
Train isn't a string it would be an entity created with `CreateMissionTrain`

Reply:
> Train isn't a string it would be an entity created with `CreateMissionTrain`

you are right, mistakenly confused this for something else, thanks!